java/src/main/java/com/logicalclocks/hsfs/FeatureGroup.java

```diff
@@ -137,6 +165,9 @@ public void save(Dataset<Row> featureData) throws FeatureStoreException, IOExcep
   public void save(Dataset<Row> featureData, Map<String, String> writeOptions)
       throws FeatureStoreException, IOException {
     featureGroupEngine.saveFeatureGroup(this, featureData, primaryKeys, partitionKeys, defaultStorage, writeOptions);
+    if (statisticsEnabled) {
+      statisticsEngine.computeStatistics(this, featureData);
+    }
```

Comment:
Shouldn't this call the `computeStatistics()` method? Otherwise you might end up computing feature for the online feature store. which is not bad per se in this case, as you are not query NDB, but might confuse users.

Reply:
I agree with the confusion part, I just wanted to reuse the dataframe as we already have it, instead of rereading it. I am not sure spark is smart enough to recognize that it's already there.

On the other hand this way it would always allow the user to have the statistics from the very first creation of the featuregroup even if it is purely online.